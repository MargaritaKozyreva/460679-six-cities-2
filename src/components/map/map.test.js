import React from "react";
import Map from "./map.jsx";
import renderer from "react-test-renderer";

describe(`Initial Map`, () => {
  const mockProps = {
    coordinates: {
      0: [52.3909553943508, 4.85309666406198],
      1: [52.369553943508, 4.85309666406198],
      2: [52.3909553943508, 4.929309666406198],
      3: [52.3809553943508, 4.939309666406198]
    }
  };
  it(`render correctly`, () => {
    const tree = renderer.create(<Map {...mockProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
