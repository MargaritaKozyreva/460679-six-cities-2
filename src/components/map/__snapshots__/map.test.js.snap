// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initial Map render correctly 1`] = `
<section
  className="cities__map map"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
